### DISCUSSION TITLE
❤️ FluxStore Manager -  Print New Order Vendors

### DISCUSSION THREAD

**Discussion by Yan-Aleman** [inspireui/support#5557] [Working on Features]
Hi team. We really need a function to automatically print receipts for new orders (and cancellations) for suppliers. Also the ability to enable and disable automatic receipt printing in the FluxStore Manager application. I think this will help make the app more popular with Multi Vendors and especially Single Vendors (woocommerce).
The ability to use the application in POS terminals like Glovo “Pos Sunmi”. Automation of the online platform could help a lot.

And in the future, support for this plugin would also not hurt;).
https://codecanyon.net/item/multipos-point-of-sale-for-wcfm-marketplace/38049472

![maxresdefault](https://user-images.githubusercontent.com/36808995/174010989-169c35cb-9683-4ffd-a275-9220b380ee8c.jpg)
![101376844_169054671249780_9223285145371410432_n](https://user-images.githubusercontent.com/36808995/174011642-d0936b2a-3f92-4ba6-8a9a-a942f4a5b8a6.jpg)

**Comment by Yan-Aleman** [comment]
@minhcasi

**Comment by weareleadsgeno** [comment]
Up

**Comment by Edmonts** [comment]
In my opinion, Fluxstore Manager needs this feature first and foremost. Printing a receipt for Vendors is a very great feature of fluxstore. As noted in the comments, it is also necessary to give vendos the opportunity to choose in which cases a check is automatically printed, according to changes in the order status. What brands of POS terminals are planned to be supported?

**Comment by osman-kocak** [comment]
Show invoice/ shipping label and download invoice/shipping label must be need on fluxmanager

**Reply by Yan-Aleman** [reply]
🔥👍
